Assets/Scripts/Tools/ToolController.cs

```diff
@@ -13,7 +13,7 @@ public class ToolController : MonoBehaviour
     ToolgunRecoil _toolgunRecoil;
     ISet<IToolChangeObserver> _toolChangeObservers;
 
-    IDictionary<KeyCode, ITool> _tools;
+    List<(KeyCode keyCode, ITool tool)> _tools;
```

Comment:
why remove dictionary functionality?

Reply:
It was a problematic data structure as there is no order to the items. It was a necessary change when I wanted to change the weapon not to a defined one, but previous or first. Doing that with dictionary would be too complex to compensate for dictionary's advantages over lists.